k8s/namespaces/unspec/demo.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: unspec-service
+spec:
+  values:
+    java:
+      environment:
+        OIDC_ISSUER: https://forgerock-am.service.core-compute-idam-demo.internal:8443/openam/oauth2/realms/root/realms/hmcts
```

Comment:
this should use the environment field in the kustomization base and only be overridden where needed, in this PR that only appears to be in aat

Reply:
It's already set please remove and only override AAT unless i'm missing something
https://github.com/hmcts/unspec-service/blob/master/charts/unspec-service/values.yaml#L39